recipe/compile_bundle.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+import ast
+import astor
+import os
+import shutil
+
+
+BAZEL_DIR = Path("utils") / "bazel"
+BAZEL_OUT = BAZEL_DIR / "bazel-bin" / "external" / "llvm-project"
+LIB_TARGET_DIR = Path(os.environ["PREFIX"]) / "lib"
+OTHER_TARGET_DIR = Path(os.environ["PREFIX"]) / "share" / "llvm_for_tf"
+
+
+def extend_linkopts(kw, name):
+    current_src = astor.to_source(kw.value)[:-1]
+    kw.value = ast.parse(f"{current_src} + ['-l{name}']").body[0].value
+
+
+def new_kwarg(name, code):
+    return ast.keyword(arg="linkopts", value=ast.parse(code).body[0].value)
+
+
+def rewrite_binaries(code, symbol):
```

Comment:
A docstring for this might be good - right now it's a bit impenetrable, though I see that we're rewriting `cc_library` calls.

I still think the argument name "symbol" is confusing, as it's used with `"LLVM"`  and `"MLIR"`, and then get's slotted into `libname`. Maybe "lib_prefix"?